torch/csrc/inductor/aoti_runtime/sycl_runtime_wrappers.h

```diff
@@ -0,0 +1,173 @@
+// NOLINT
+#pragma once
+#ifdef USE_XPU
+#include <level_zero/ze_api.h>
+#include <sycl/sycl.hpp>
+#include <fstream>
+#include <iostream>
+#include <string>
+
+#define ZE_CHECK(status)                                                  \
+  {                                                                       \
+    if (status != ZE_RESULT_SUCCESS) {                                    \
+      std::stringstream ss;                                               \
+      ss << "L0 runtime error: " << std::hex << std::uppercase << status; \
+      throw std::runtime_error(ss.str());                                 \
+    }                                                                     \
+  }
+
+static ze_module_handle_t create_module(
+    ze_context_handle_t context,
+    ze_device_handle_t device,
+    const uint8_t* binary_ptr,
+    size_t binary_size) {
+  const char* build_flags = "";
+  const ze_module_format_t format = ZE_MODULE_FORMAT_IL_SPIRV;
```

Comment:
We'll change the format to binary after Triton update with binary kernel feature, and will not support load SPIR-V in AOTI.

> The `format` should be a parameter of this function rather than a hardcode `ZE_MODULE_FORMAT_IL_SPIRV`. Because we need to support device binary but not only SPIRV.

@EikanWang :  We'll change the format to binary after Triton update with binary kernel feature, and will not support load SPIR-V in AOTI.

Reply:
resolved.